docs/MorganeyIndex.md

```diff
@@ -0,0 +1,32 @@
+# Morganey Index
+
+Morganey index is a file called `morganey-index` that is located at
+the root of Morganey module container. Morganey module container is a
```

Comment:
I suggest that you rephrase that as "the root of _every_ Morganey module container".